docs/content/preview/contribute/core-database/build-from-src-centos.md

```diff
@@ -56,7 +65,8 @@ sudo yum install -y centos-release-scl epel-release git libatomic rsync which
 
 ### Python 3
 
-Python 3.7 or higher is required.
+{{% readfile "includes/python.md" %}}
+
 The following example installs Python 3.8.
```

Comment:
You're using 3.9 in the Alma instructions...

Reply:
I couldn't find a higher python version using the package manager.  See https://github.com/yugabyte/yugabyte-db/pull/16074/commits/b07a247b938750fe19f533a69746acefa7741af8 for in-progress work building python from source before I realized 3.8 existed in an additional package repository.

Reply:
Ugh that sucks! 3.8 sounds lovely here :)